projects/valtimo/dossier/src/lib/services/dossier-list-case-tag.service.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Injectable} from '@angular/core';
+import {DossierListService} from './dossier-list.service';
+import {CaseTagService, CaseTag} from '@valtimo/document';
+import {DossierParameterService} from './dossier-parameter.service';
+import {BehaviorSubject, combineLatest, map, Observable, switchMap, take, tap} from 'rxjs';
+
+@Injectable()
+export class DossierListCaseTagService {
+  private readonly _selectedCaseTags$ = new BehaviorSubject<CaseTag[]>([]);
+
+  private readonly _showCaseTagsSelector$ = new BehaviorSubject<boolean>(false);
+
+  private readonly _caseTags$: Observable<Array<CaseTag>> =
```

Comment:
I know it's the same thing, but it's not consisted with what you've written so far

```suggestion
  private readonly _caseTags$: Observable<CaseTag[]> =
```

Reply:
Removed array